src/version.hpp

```diff
@@ -0,0 +1,97 @@
+/*!
+A constexpr way to obtain program version information.
+Copyright © 2018, Viktor Zoutman
```

Comment:
... and team? 😭

Reply:
:| I wrote this code a years ago forgot to update the license.